examples/lib/defaultHtmlStepUi.js

```diff
@@ -43,19 +53,47 @@ function DefaultHtmlStepUi(_sequencer, options) {
     </div>\
     </div>\
     </div>\
+    <div class="panel-footer">\
+        <button class="btn btn-primary"> Apply\
+        <button class="btn  btn-primary pull-right insert-step">\
+          <i class="fa fa-plus"></i> Insert module\
+        </button>\
+      </div>\
+    </div>\
     </div>';
 
-    var tools =
-    '<div class="cal"><div class="tools btn-group">\
-    <button confirm="Are you sure?" class="remove btn btn btn-default">\
-      <i class="fa fa-trash"></i>\
-    </button>\
-    <button class="btn  insert-step" style="margin-left:10px;border-radius:6px;background-color:#fff;border:solid #bababa 1.1px;" >\
-      <i class="fa fa-plus"></i> Add\
-    </button>\
+    step.load=
```

Comment:
Ah can you please check out the proper way to do card UI HTML in the Bootstrap docs here? https://getbootstrap.com/docs/3.3/components/#panels-heading This should help tidy things up! And the dark line under the module name can be removed since we're going to just display it in the card header now. Thanks!

Reply:
can you say me how to remove the dark line under the module as i can't do it

Reply:
What was the change do I need to do it here??